packages/expo-navigation-bar/src/NavigationBar.ts

```diff
@@ -139,219 +45,33 @@ export async function getBorderColorAsync(): Promise<string> {
- * - `'inset-touch'`: Reveals the System UI after a touch anywhere on the screen and insets your App's content (Safe Area). The System UI is visible until you explicitly hide it again.
- *
- * @example
- * ```ts
- * await NavigationBar.setBehaviorAsync('overlay-swipe')
- * ```
- * @param behavior Dictates the interaction behavior of the navigation bar.
- */
-export async function setBehaviorAsync(behavior: NavigationBarBehavior): Promise<void> {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
-  if (Platform.OS !== 'android') {
-    console.warn('`setBehaviorAsync` is only available on Android');
-    return;
-  }
-  return await ExpoNavigationBar.setBehaviorAsync(behavior);
-}
-
-/**
- * Gets the behavior of the status and navigation bars when the user swipes or touches the screen.
- *
- * @example
- * ```ts
- * await NavigationBar.getBehaviorAsync()
- * ```
- * @returns Navigation bar interaction behavior. Returns `inset-touch` on unsupported platforms (iOS, web).
- */
-export async function getBehaviorAsync(): Promise<NavigationBarBehavior> {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
-  if (Platform.OS !== 'android') {
-    console.warn('`getBehaviorAsync` is only available on Android');
-    return 'inset-touch';
-  }
-  return await ExpoNavigationBar.getBehaviorAsync();
-}
-
-/**
- * React hook that statefully updates with the visibility of the system navigation bar.
- *
- * @example
- * ```ts
- * function App() {
- *   const visibility = NavigationBar.useVisibility()
- *   // React Component...
- * }
- * ```
- * @returns Visibility of the navigation bar, `null` during async initialization.
- */
-export function useVisibility(): NavigationBarVisibility | null {
-  if (__DEV__) {
-    potentiallyWarnAboutEdgeToEdge();
-  }
-
-  const [visibility, setVisible] = useState<NavigationBarVisibility | null>(null);
-
-  useEffect(() => {
-    let isMounted = true;
-    if (Platform.OS !== 'android') {
-      setVisible('hidden');
-      return;
-    }
-    getVisibilityAsync().then((visibility) => {
-      if (isMounted) {
-        setVisible(visibility);
-      }
-    });
-
-    const listener = addVisibilityListener(({ visibility }) => {
-      if (isMounted) {
-        setVisible(visibility);
-      }
-    });
-
-    return () => {
-      listener.remove();
-      isMounted = false;
-    };
-  }, []);
-
-  return visibility;
-}
-
-export * from './NavigationBar.types';
+// MARK: existing expo-navigation-bar APIs
+
+export {
+  addVisibilityListener,
+  setBackgroundColorAsync,
+  getBackgroundColorAsync,
+  setBorderColorAsync,
+  getVisibilityAsync,
+  setButtonStyleAsync,
+  getButtonStyleAsync,
+  setPositionAsync,
+  unstable_getPositionAsync,
+  setBehaviorAsync,
+  getBehaviorAsync,
+  useVisibility,
+  getBorderColorAsync,
+} from './NativeNavigationBarWrapper';
```

Comment:
```suggestion
export * from './NativeNavigationBarWrapper';
```

Q: Can we export everything from the file, instead listing methods? Usually setups like that are often error-prone with time, and could cause missing types issues in docs.

If there is anything that need to be exported from `NativeNavigationBarWrapper` but should not be exported here probably can be move out to a separate file, which can be imported only in wrapper file.